tests/lru_cache_test.py

```diff
@@ -49,25 +49,25 @@ class LRUCacheTest(LRUCacheTestCase):
 
   def test_get_nonexistent_key(self):
     cache = LRUCache(self.name, max_size=-1)
-    self.assertIsNone(cache.get("cache-a"))
+    self.assertIsNone(cache.get("a"))
 
   def test_put_and_get_key(self):
     cache = LRUCache(self.name, max_size=-1)
 
-    cache.put("cache-a", b"a")
-    self.assertEqual(cache.get("cache-a"), b"a")
-    self.assertEqual(set(self.path.glob("cache-*")), {self.path / "cache-a"})
+    cache.put("a", b"a")
+    self.assertEqual(cache.get("a"), b"a")
+    self.assertEqual(set(self.path.glob(f"*{cache._cache_suffix}")), {self.path / f"a{cache._cache_suffix}"})
```

Comment:
This seems very verbose. I am thinking about moving the logic into a separate function, but I don't know how the `LRUCacheTestCase` class works, and I am unsure if I can just create a member function of the `LRUCacheTestCase` class.

Reply:
You can create a new assertion method in `LRUCacheTestCase` if you want, but I think this if fine, actually.